[2023-05-20T15:00:15.478+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv2.postgres_to_s3_task scheduled__2023-05-19T15:00:00+00:00 [queued]>
[2023-05-20T15:00:15.489+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv2.postgres_to_s3_task scheduled__2023-05-19T15:00:00+00:00 [queued]>
[2023-05-20T15:00:15.489+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-20T15:00:15.490+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-20T15:00:15.490+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-20T15:00:15.513+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): postgres_to_s3_task> on 2023-05-19 15:00:00+00:00
[2023-05-20T15:00:15.518+0000] {standard_task_runner.py:55} INFO - Started process 153 to run task
[2023-05-20T15:00:15.525+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv2', 'postgres_to_s3_task', 'scheduled__2023-05-19T15:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv2.py', '--cfg-path', '/tmp/tmptqyd9lgc']
[2023-05-20T15:00:15.525+0000] {standard_task_runner.py:83} INFO - Job 85: Subtask postgres_to_s3_task
[2023-05-20T15:00:15.606+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv2.postgres_to_s3_task scheduled__2023-05-19T15:00:00+00:00 [running]> on host 2e718862c733
[2023-05-20T15:00:15.710+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv2
AIRFLOW_CTX_TASK_ID=postgres_to_s3_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-19T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-19T15:00:00+00:00
[2023-05-20T15:00:15.726+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-20T15:00:15.731+0000] {todays_gamesv2.py:132} INFO - Ran this on 2023-05-19. Saved postgres data in text file: games.txt
[2023-05-20T15:00:15.740+0000] {base.py:73} INFO - Using connection ID 'aws_hook' for task execution.
[2023-05-20T15:00:15.741+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_hook', conn_type='aws') credentials retrieved from login and password.
[2023-05-20T15:00:17.925+0000] {todays_gamesv2.py:141} INFO - Test data file /tmp/tmpr3314bxv has been pushed to S3
[2023-05-20T15:00:17.927+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-20T15:00:17.938+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=get_todays_gamesv2, task_id=postgres_to_s3_task, execution_date=20230519T150000, start_date=20230520T150015, end_date=20230520T150017
[2023-05-20T15:00:17.983+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-20T15:00:18.018+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
